src/browser/MSALAuthenticationProvider.ts

```diff
@@ -25,40 +25,33 @@ declare const Msal: any;
  * @extends AuthenticationProvider
  */
 export class MSALAuthenticationProvider implements AuthenticationProvider {
-	/**
-	 * @private
-	 * A member holding the clientId of an application
-	 */
-	private clientId: string;
-
 	/**
 	 * @private
 	 * A member holding the list of graph scopes
 	 */
 	private scopes: string[];
 
 	/**
-	 * @private
+	 * @public
 	 * A member holding an instance of UserAgentApplication returned from MSAL
 	 */
-	private userAgentApplication: any;
+	public userAgentApplication: any;
 
 	/**
 	 * @public
 	 * @constructor
 	 * Creates an instance of MSALAuthenticationProvider
-	 * @param {string} clientId - The clientId value of an application
+	 * @param {string | any} clientIdOrUserAgentApplication - The clientId value of an application or an instance of UserAgentApplication
 	 * @param {string[]} scopes - An array of graph scopes
 	 * @param {any} [options] - An options object for MSAL initialization
 	 * @returns An instance of MSALAuthenticationProvider
 	 */
-	public constructor(clientId: string, scopes: string[], options?: any) {
+	public constructor(clientIdOrUserAgentApplication: string | any, scopes: string[], options?: any) {
 		const callback = (errorDesc, token, error, tokenType) => {
```

Comment:
If we don't do anything in this callback, what do we do with errors that are returned when attempting to get a token?